src/js/main.js

```diff
@@ -399,7 +399,20 @@
 
             render() {
                 cx.fillStyle = 'cyan';
-                cx.fillRect(this.x, this.y, this.width, this.height);
+                // Different size for drawing than for collision checking.
+                let w = this.width * 1.2, h = this.height * 1.1;
+                let x = this.x - (w - this.width) * 0.5;
+                let y = this.y - (h - this.height) * 0.7 + 4;
```

Comment:
Is there some reasoning for this 0.7 + 4, instead of just 0.5 like for the width? With ghosts it's a different thing because they are rounded from the top, so the collision rectangle is a bit lower (I could have commented that part).

Also, do you think that a smaller collision rectangle is needed here? The ghosts already have a smaller collision rectangle, so colliding with them doesn't happen so easily. Now from what I tried, the player can overlap quite a bit with the ghosts (the ghost being below) without hitting. In addition, when the player is a rectangle, the collision box can be exactly the same as the player shape, so there's no need to make it smaller necessarily.